tests/python/VultureBearTest.py

```diff
@@ -42,85 +42,115 @@ def prep_file():
 
             return list(self.uut.run())
 
+    def verify_results(self, test_case, expected):
+        detected = dict((item.message, (item.affected_code[0].start.line,
+                                        item.affected_code[0].end.line,
+                                        item.confidence))
+                        for item in self.get_results(test_case))
+        self.assertEqual(detected, expected)
```

Comment:
@Makman2 IMHO `VultureBear` doesn't needs to be rigorously tested here. All it does is pass on values from vulture. Can we open a different issue and then change this afterwards?